app/models/king.rb

```diff
@@ -4,4 +4,78 @@ def valid_move?(x, y)
     columns = (x - x_position).abs
     rows <= 1 && columns <= 1 ? true : false
   end
+
+  def castle!(rook)
+    fail "King cannot currently be castled!" unless can_castle?(rook)
```

Comment:
I'm thinking we should change this to a flash alert message too, probably easier to just drop the whole failing paradigm....in fact i think we should have this method return `true` if the castle happened and `false` if it didn't....that way it can be used more easily on the front end.